test/inductor/test_inplace_padding.py

```diff
@@ -117,6 +118,7 @@ def f(x, y):
 
         self.assertEqual(num_inplace_padding(), 0)
 
+    @requires_cuda
```

Comment:
It's simpler to just change 
```
    @unittest.skipIf(
        torch.cuda.is_available()
        and torch.cuda.get_device_properties().total_memory < 2e10,
        "Only if the GPU has at least 20GB memory to be safe",
    )
```
to
```
    @unittest.skipIf(
        not torch.cuda.is_available()
        or torch.cuda.get_device_properties().total_memory < 2e10,
        "Only if the GPU has at least 20GB memory to be safe",
    )
```

The PR is reverted due to breaking some meta internal tests. I'll include the fix in a reland.

Reply:
Greate, thanks.